next.config.js

```diff
@@ -1,5 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  images: {
+    domains: ["cdn.sanity.io", "pbs.twimg.com", "f005.backblazeb2.com"],
```

Comment:
lol what are these domains? 😂 

non-blocking they just feel hella random

Reply:
pbs.twimg.com is twitter user profile image links, f005.backblazeb2.com is where all the teams images are stored, I didnt want to bloat the public folder with image assets. I forget what cdn.sanity.io is for 🤣